src/Cake.Issues/IIssueComparer.cs

```diff
@@ -0,0 +1,135 @@
+﻿namespace Cake.Issues
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Comparer to compare if two issues are identical.
+    /// </summary>
+    public class IIssueComparer : IEqualityComparer<IIssue>
```

Comment:
Unlovely that the class name begins with an `I` which is usually used for interfaces. But I don't have any better suggestion for this :-)